tests/rules/test_indentation.py

```diff
@@ -1369,6 +1369,12 @@ def test_nested_collections(self):
                    '     - - - - - very nested: a\n'
                    '               key: value\n'
                    '...\n', conf, problem=(2, 2))
+        conf = 'indentation: {indent-sequences: true}'
+        self.check('---\n'
+                   '- item:\n'
+                   '  - elem 1\n'
+                   '  - elem 2\n'
+                   '...\n', conf, problem=(3, 3))
```

Comment:
Could you move this new test case in a dedicated function, e.g. `test_nested_collections_with_spaces_consistent()`, and add a small link to `https://github.com/adrienverge/yamllint/issues/485` so we easily understand what it's about and avoid touching it too much?

Reply:
Sure, I just pushed an amended commit with the test moved to its own function with a reference to the issue along with additional tests to include the other possible indent-sequences values.